### PR TITLE
Added contacts phone numbers update method

### PR DESCRIPTION
**Pull request by Tal-D**

### CODE REVIEW

**Comment by georgettica** [comment]
Thanks for moving it here :)

File: quickstart.rb [hunk 1]
```diff
@@ -63,9 +63,16 @@ def init_service
 
 
 first_filter = cli.ask("What filter do you want ot run?  ") { |q| q.default = "[\+|0-9][0-9|\s|\\-|a-z|A-Z]*" }
+replacement_pattern = cli.ask("Choose replacement pattern (optional)  ") { |q| q.default = "\\0" }
```
Comment (Tal-D) [inline quickstart.rb:66]: Could think about a good query statement for the replacement pattern 🤷‍♂️

File: spec/contact_fixer_spec.rb [hunk 2]
```diff
@@ -78,4 +78,45 @@
       end
     end
   end
+  describe '.update_connections_phone_numbers' do
+    before(:each) do
+      @replacement_pattern = '123'
+      @contact_number = "0118-999-881-999-119-725-3"
```
Comment (Tal-D) [inline spec/contact_fixer_spec.rb:84]: Reference to https://theitcrowd.fandom.com/wiki/0118_999_881_99_9119_7253

File: quickstart.rb (hunk 1 above)
Comment (georgettica) [reply quickstart.rb:66]: The default replace can be an empty one, which would do the trick, Right?

This way a no-op is guaranteed

File: spec/contact_fixer_spec.rb (hunk 2 above)
Comment (georgettica) [reply spec/contact_fixer_spec.rb:84]: Hahaha, didn't see that episode yet

**Comment by georgettica** [comment]
Is this PR ready to review?

File: quickstart.rb (hunk 1 above)
Comment (Tal-D) [reply quickstart.rb:66]: When adding "\\\\0" as the replacement, gsub method replaces the matching expression with itself (expression with index=0 flag). I repeated '\\' twice to include '\\' as a character in the default pattern.

**Comment by Tal-D** [comment]
Yes - the relevant tests were added.

**Pull request merged**